readthedocs/profiles/views.py

```diff
@@ -109,9 +110,7 @@ def create_profile(
     return render(request, template_name, context=context)
 
 
-create_profile = login_required(create_profile)
-
-
+@login_required
```

Comment:
Don't we need `@login_required()` with `()` here? Or maybe django handle both cases? Could you link to the django docs?

Reply:
We don't need `()` for the decorators.
https://docs.djangoproject.com/en/2.1/topics/auth/default/#the-login-required-decorator